educator/tests/Test/Dscp/Educator/Web/Educator/Queries.hs

```diff
@@ -204,34 +206,6 @@ spec_Educator_API_queries = specWithTempPostgresServer $ do
                 ===
                 sortOn fst submissionsAndGrades'
 
-        it "Filtering works" $ sqlPropertyM $ do
-            env <- pickSmall $ genCoreTestEnv simpleCoreTestParams
-                                         { ctpAssignment = variousItems }
-            courseIdF <- pick arbitrary
-            assignHF <- pick arbitrary
-            docTypeF <- pick arbitrary
-
-            let student = tiOne $ cteStudents env
-                sigSubs = ordNub . tiList $ cteSignedSubmissions env
-                courses = map _aCourseId . tiList $ cteAssignments env
-
-            lift $ prepareForSubmissions env
-            lift $ mapM_ createSignedSubmission sigSubs
-
-            submissions <- lift $ educatorGetSubmissions
-                def{ sfStudent = Just student, sfCourse = courseIdF
-                    , sfAssignmentHash = assignHF, sfDocType = docTypeF }
-                def
-
-            let submissions' =
-                  map (\(s, _) -> educatorLiftSubmission s Nothing) $
-                  applyFilterOn snd courseIdF $
-                  applyFilterOn (_sAssignmentHash . _ssSubmission . fst) assignHF $
-                  applyFilterOn (_sDocumentType . _ssSubmission . fst) docTypeF $
-                  sigSubs `zip` cycle courses
-
-            return $ sortOn siHash submissions === sortOn siHash submissions'
-
```

Comment:
What's wrong with this test?

Reply:
I think at the current level of automatization there is no need to test filtering as soon as many human mistakes are now impossible.
There is one test which covers filtering on `DocType`, and if there will be other complex filters I will write tests on them as well.

For me, it looks like we already have too many tests which are costly to maintain (each endpoint change requires fixing multiple test cases), so I'd like to avoid an unnecessary test code.